internal/build/imgsrc/docker.go

```diff
@@ -157,7 +159,14 @@ func (t DockerDaemonType) PrefersLocal() bool {
 }
 
 func NewLocalDockerClient() (*dockerclient.Client, error) {
-	c, err := dockerclient.NewClientWithOpts(dockerclient.WithAPIVersionNegotiation())
+	opts := []dockerclient.Opt{dockerclient.WithAPIVersionNegotiation()}
+
+	host, err := exec.Command("docker", "context", "inspect", "-f", "{{ .Endpoints.docker.Host }}").Output()
+	if err == nil {
+		opts = append(opts, dockerclient.WithHost(strings.TrimSpace(string(host))))
```

Comment:
Odd to see TrimSpace here. Does it return something like `   hostname   `?

Reply:
Yes. Hostname plus trailing newline.